app/helpers/application_helper.rb

```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def format_date(time)
+    time.strftime('%A, %B %d, %Y')
+  end
```

Comment:
Woo! Thanks for adding this helper method for formatting dates! Super helpful

Reply:
Yes, this looks great!